### DISCUSSION TITLE
Cannot enable paging for Messages entity type in Microsoft Search

### DISCUSSION THREAD

**Discussion by ThomasVVK** [microsoft-search/pnp-modern-search#2349] [Q&A]
**Version used**
v4.6.1

**Describe the bug**
Hi,

I have added a search result webpart displaying mail from Outlook. I would like to paginate this list of results to optimize it display.
Despite having enabled navigation buttons, I cannot see them. I tried to edit the results template to try to detect any track. I have been able to make the buttons visible by removing the lines  hilighted in red in my screenshots and by turning the option "Hide if disabled" off (see in the second screenshot). 

I have also added a screenshot with paging properties.

Ps: I am a novice at this, I am very sorry if I don't give the appropriate details!

**To Reproduce**
1. Go to any communication site
2. Add a Search Results Webpart
3. Link it to Microsoft Search: entity type: Messages
4. Enable paging with items per page < total number of items

**Expected behavior**
Visible and functional navigation buttons

**Screenshots**
![EditCode](https://user-images.githubusercontent.com/111756510/188826352-de9ba74e-7404-4754-9af9-eeda64c4a096.png)
![NavigationSetttings](https://user-images.githubusercontent.com/111756510/188826360-4ceee41e-9929-4bca-baac-173c469e52b2.png)
![PagingProperties](https://user-images.githubusercontent.com/111756510/188826366-5e6105bc-12a9-4530-a478-69cc7c5dab68.png)

**Desktop (please complete the following information):**
 - Browser: Chrome

**Additional context**
Thank you very much for your help!

**Comment by FranckyC** [comment]
Hi @ThomasVVK, quick question: how many results do you get? If this number is below the default number of items, it is the excepted behavior to hide the pagination control. That's the condition do acutally

**Comment by ThomasVVK** [comment]
Hi @FranckyC , 
Thank you for deeping into this!

I have in total 25 results for my search and I am dispplaying 5 items per page but the navigation buttons still don't show up.
